src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\DataCollector\RouterDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\AjaxDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\ConfigDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\EventDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\ExceptionDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\LoggerDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\MemoryDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\RequestDataCollector;
+use Symfony\Component\HttpKernel\DataCollector\TimeDataCollector;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('data_collector.config', ConfigDataCollector::class)
+            ->call('setKernel', [service('kernel')->ignoreOnInvalid()])
+            ->tag('data_collector', ['template' => '@WebProfiler/Collector/config.html.twig', 'id' => 'config', 'priority' => -255])
+
+        ->set('data_collector.request', RequestDataCollector::class)
+            ->tag('kernel.event_subscriber')
+            ->tag('data_collector', ['template' => '@WebProfiler/Collector/request.html.twig', 'id' => 'request', 'priority' => 335])
+
+        ->set('data_collector.request', AjaxDataCollector::class)
```

Comment:
`data_collector.ajax`

Reply:
Aaargh, thx :].

Will the tests pick this up? Locally running `php ./phpunit src/Symfony/Bundle/FrameworkBundle` did not report any errors. (Or is it in another test?)

Reply:
Fixed it, was just wondering.